getsounds.py

```diff
@@ -0,0 +1,131 @@
+#all_sound_ids = json.load(open(PATH_TO_ALL_SOUND_IDS,'rb'))
+
+
+
+def compile_annotated_sounds(annotations_path):
+    annotated_sounds = {}
+    
+    annotation_files = glob.glob(annotations_path + '/*/*.json', recursive=True)
+    for an_file in annotation_files:
+        an = json.load(open(an_file,'rb'))
+        #remove the path, then the extension and then the "sound-" string
+        sound_id = os.path.basename(os.path.splitext(an_file)[0]).replace('sound-','')
+        if sound_id in annotated_sounds:
+            annotated_sounds[sound_id] = {  "genres" : list(set(an["genres"] + annotated_sounds[sound_id]["genres"])),
+                                            "num_annotations" : annotated_sounds[sound_id]["num_annotations"] + 1   }
+        else:
+            annotated_sounds[sound_id] = {  "genres" : an["genres"],
+                                            "num_annotations" : 1   }
+
+    return annotated_sounds
+
+
+def collect_authors(sounds_annotated,metadata):
+    author_sounds = {}
+    for sound in sounds_annotated:
+        author = metadata[sound]["username"]
+        if author in author_sounds.keys():
+            author_sounds[author] = author_sounds[author] + 1
+        else:
+            author_sounds[author] = 1
+        
+    return author_sounds
+
+
+def collect_packs(sounds_annotated,metadata):
+    pack_sounds = {}
+    for sound in sounds_annotated:
+        pack = metadata[sound]["pack_name"]
+        if pack is not None:
+            if pack in pack_sounds.keys():
+                pack_sounds[pack] = pack_sounds[pack] + 1
+            else:
+                pack_sounds[pack] = 1
+        
+    return pack_sounds
+
+def collect_genres(sounds_annotated):
+    genre_sounds = {}
+    for sound in sounds_annotated:
+        genres = sounds_annotated[sound]["genres"]
+        for genre in genres:
+            if genre in genre_sounds.keys():
+                genre_sounds[genre] = genre_sounds[genre] + 1
+            else:
+                genre_sounds[genre] = 1
+        
+    return genre_sounds
+    
+def genre_importance(sound_genres, genre_sounds):
+    importance = 0
+    genre_sounds_sorted = sorted(genre_sounds.items(), key=lambda x: x[1])
+    less_annotated_sounds = [genre_sounds_sorted[0][0], genre_sounds_sorted[1][0]]
+    for genre in sound_genres:
+        if genre in less_annotated_sounds:
+            importance = importance + 1
+    
+    return importance
+
+def discard_packs(all_sound_ids,metadata):
+    packs_to_discard = []
+    for sound in all_sound_ids:
+        if metadata[sound]["pack"] in packs_to_discard:
+            all_sounds_ids.remove(sound)
+
+    return all_sound_ids
+
+
+
+def select_relevant_sounds(annotations_path, metadata, genre_metadata, all_sound_ids, N=default_N_assign_more_sounds):
+
+    sounds_annotated = compile_annotated_sounds(annotations_path)
+
+    anno_weight=1000
+    auth_weight=1
+    pack_weight=1
+    genre_weight=-10
+
```

Comment:
You can add a comment before each of the weight definition to explain what does it mean and why we set it that way